checkout/application/placeorder/coordinator.go

```diff
@@ -60,10 +63,17 @@ var (
 	ErrAnotherPlaceOrderProcessRunning = errors.New("ErrAnotherPlaceOrderProcessRunning")
 
 	maxLockDuration = 2 * time.Minute
+
+	// newUUIDCount counts generation of new UUIDs
+	newUUIDCount = stats.Int64("flamingo/standard_place_order/new_uuid", "Count of logs", stats.UnitDimensionless)
```

Comment:
To match the other exposed metrics I would suggest changing the namespace to something like:
`flamingo-commerce/checkout/placeorder/`

In addition this metric isn't really speaking, yes there will be a new uuid but that doesn't express what that means. In this case a new place order process is started. So i would rename from `new_uuid` to `new` or `starts`. What is the second parameter used for `"Count of logs"`?

Reply:
Removed the description now on all three metrics.